test/cmd.js

```diff
@@ -1014,6 +1014,128 @@ describe('express(1)', function () {
         })
       })
     })
+
+    describe('es2015', function () {
```

Comment:
@imcodingideas / @dougwilson, if you want to help: The only issue I know about, is that the `--es2015` flag test also runs in older Node environments in the CI. Because older versions doesn't support es2015, the tests are failing.

We should either: 
- Test es2015 support even when the `--es2015` flag is presented. 
- Disable this test if it runs on older Node versions. 

Personally I think it's obvious that the first option is the best option.

If you'd like to help, feel free to discuss it and clear out what exactly should be done to solve this (@dougwilson), or just go ahead and implement it yourselves. 
That said, I might jump in and just do it myself if I'll find the time :) 

Thanks!

Reply:
OK. I like the test es2015 support when you use the flag. I'm just seeing these messages. I've had a long month.